ArcGISRuntimeSDKQt_CppSamples/Layers/GroupLayers/README.md

```diff
@@ -0,0 +1,38 @@
+# Group layers
+
+Group a collection of layers together and toggle their visibility as a group.
+
+![](screenshot.png)
+
+## Use case
+
+Group layers communicate to the user that layers are related and can be managed together.
+
+In a land development project, you might group layers according to the phase of development.
+
+## How to use the sample
+
+The layers in the map will be displayed in a table of contents. Toggle the checkbox next to a layer's name to change its visibility. Turning a group layer's visibility off will override the visibility of its child layers.
+
+## How it works
+
+1. Create an empty `GroupLayer`.
+2. Add a child layer to the group layer's layers collection.
```

Comment:
Should this explanation go the other way around? It's not a child layer to begin with since you're building up the group layer from scratch.

Maybe: `Add the new layer to the group layer's layers collection so it will be a child layer.`, or something like that?